arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -9194,12 +9190,7 @@
           "$ref": "#/definitions/FabricSpecificUpdateNetworkMappingInput"
         }
       ],
-      "properties": {
-        "InstanceType": {
-          "description": "The instance type.",
-          "type": "string"
-        }
-      },
+      "properties": {},
```

Comment:
@jhendrixMSFT model VmmToAzureUpdateNetworkMappingInput in service as of now doesn't have any value specific to this instance as of now and has been introduced to make it future extensible and consistent with other sibling models (we expect it to have some fields in near future).
Swagger changes just represent what the service reflects

Swagger and changes in swagger exa